dockerfile_parse/util.py

```diff
@@ -188,3 +188,113 @@ def remove_nonescaped_quotes(string):
     string = remove_quotes(string)
     string = string.replace('\ ', ' ')
     return string.replace("\\s", "'").replace('\\d', '"')  # restore
+
+
+def split_tuple(text):
+    text_split = text.split('=')
```

Comment:
Thanks. Would you mind also adding a test case for that? It's a shame it wasn't caught by unit tests.